src/target/riscv/riscv.c

```diff
@@ -2147,10 +2198,12 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.name = "use_bscan_tunnel",
 		.handler = riscv_use_bscan_tunnel,
 		.mode = COMMAND_ANY,
-		.usage = "riscv use_bscan_tunnel value",
+		.usage = "riscv use_bscan_tunnel value {type,optional}",
```

Comment:
Replace "{type, optional}" with "[type]".
The square brackets indicate the argument is options.

Reply:
https://github.com/open-source-forks/riscv-openocd/blob/74b0b3046fc56a9d404eb8ef3352b3f5dcd6bd45/src/target/riscv/riscv.c#L2197-L2207